aries/aries_vcx_ledger/src/ledger/base_ledger.rs

```diff
@@ -85,16 +93,35 @@ pub trait AnoncredsLedgerRead: Debug + Send + Sync {
         cred_def_id: &CredentialDefinitionId,
         submitter_did: Option<&Did>,
     ) -> VcxLedgerResult<CredentialDefinition>;
+
+    /// Get the anoncreds [RevocationRegistryDefinition] associated with the given ID.
+    /// Also returns any trait-specific additional metadata for the rev reg.
     async fn get_rev_reg_def_json(
         &self,
         rev_reg_id: &RevocationRegistryDefinitionId,
-    ) -> VcxLedgerResult<RevocationRegistryDefinition>;
+    ) -> VcxLedgerResult<(
+        RevocationRegistryDefinition,
+        Self::RevocationRegistryDefinitionAdditionalMetadata,
+    )>;
     async fn get_rev_reg_delta_json(
```

Comment:
add deprecated flag

Reply:
@JamesKEbert i added this flag in my recent commit, you'll see i had to add `#[allow(deprecated)]` in all places where it's still being used - this makes it a bit clearer where it's still being used